.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13.x
```

Comment:
Can you change this to 1.15.x to match the travis build? (I can also do that after merge)